include/my_global.h

```diff
@@ -28,7 +28,9 @@
 #include "my_config.h"
 
 #define __STDC_LIMIT_MACROS	/* Enable C99 limit macros */
+#ifndef __STDC_FORMAT_MACROS
 #define __STDC_FORMAT_MACROS	/* Enable C99 printf format macros */
+#endif /* __STDC_FORMAT_MACROS */
 #define _USE_MATH_DEFINES       /* Get access to M_PI, M_E, etc. in math.h */
```

Comment:
Why is this changed? This is in server code and will require a launchpad bug or blueprint explaining the change.